README.md

```diff
@@ -93,146 +47,93 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- Add a new game, a music album and a book
+- Add extra details such genre, label and authors
+- All the data is persisted in JSON files
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
+- A text editor preferrably [Visual Studio code](https://code.visualstudio.com/)
+- Latest version of [Ruby](https://www.ruby-lang.org/en/downloads/) installed
+- [PostgreSQL Server](https://www.postgresql.org/download/)
 
-```sh
- gem install rails
-```
- -->
 
 ### Setup
 
-Clone this repository to your desired folder:
-
-<!--
-Example commands:
+Clone this [repository](https://github.com/Fayob/catalog_of_my_things) to your desired folder:
 
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone https://github.com/Fayob/catalog_of_my_things.git
+  cd catalog_of_my_things
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
   gem install
 ```
---->
 
 ### Usage
 
-To run the project, execute the following command:
-
-<!--
-Example command:
+To run the project, execute the following command in your termainal:
 
 ```sh
-  rails server
+  ruby main.rb
```

Comment:
- [ ] [OPTIONAL] Kindly note that the `main.rb` file is embedded in the /SRC directory so the right command to launch this app is supposed to be :
```
ruby src/main.rb
```
Let's implement this in order to have a better user experience(UX).

Reply:
Thanks for the feedback